searchcore/src/vespa/searchcore/proton/bucketdb/bucket_db_owner.h

```diff
@@ -44,6 +44,7 @@ class BucketDBOwner
     Guard takeGuard() {
         return Guard(&_bucketDB, _mutex);
     }
+    const BucketDB & getBucketDB() const { return _bucketDB; }
```

Comment:
This method should not be added.

Reply:
No, and it was not used either. Gone now.